lessons/06-css-development-style/README.md

```diff
@@ -449,4 +449,4 @@ $ npm i --save lodash
 
 and then use it in Node: ```var _ = require('lodash');```
 
-Read through the [lodash docs](https://lodash.com/docs) (or check out their [searchable interfacte](http://devdocs.io/lodash-array/)) to get an idea of some of the useful helper functions
\ No newline at end of file
+Read through the [lodash docs](https://lodash.com/docs) (or check out their [searchable interfacte](http://devdocs.io/lodash-array/)) to get an idea of some of the useful helper functions
```

Comment:
Well, since it's flagging it anyway, you may as well fix the spelling of "interface" here as well :P.